debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -84,7 +84,7 @@
 @ThreadSafe
 public final class EmbeddedEngine implements DebeziumEngine<SourceRecord>, EmbeddedEngineConfig {
 
-    public static final class BuilderImpl implements Builder {
+    public static final class BuilderImpl implements Builder<SourceRecord> {
```

Comment:
What do you think to rename to `EngineBuilder`?

Reply:
@mfvitale good idea, thanks.